src/templates/controller/getPaths.ts

```diff
@@ -10,8 +11,20 @@ export const getPaths = (
     serviceName: string,
 ) => {
     const { parameters, requestBody, responses } = data
-    const pathParams = parameters?.filter((param) => param.in === paramTypes.PATH).map(({ name }) => name)
-    const queryParams = parameters?.filter((param) => param.in === paramTypes.QUERY).map(({ name }) => name)
+    const pathParams: { name: string; schema: RefSchema }[] = []
+    const queryParams: { name: string; schema: RefSchema }[] = []
+
+    if (parameters) {
+        for (const param of parameters) {
+            if (param.in === paramTypes.PATH) {
+                pathParams.push({ name: param.name, schema: param.schema })
+            }
+            if (param.in == paramTypes.QUERY) {
+                queryParams.push({ name: param.name, schema: param.schema })
+            }
+        }
```

Comment:
```suggestion
        for (const param of parameters) {
           const { name, schema } = param
            if (param.in === paramTypes.PATH) {
                pathParams.push({ name, schema})
            }
            if (param.in == paramTypes.QUERY) {
                queryParams.push({ name, schema })
            }
        }
```

или так как параметры могут быть только одним из двух вариантов (но так редко пишут. редко видел такое):

```suggestion
parameters.forEach(({name, schema, ...rest}) => (rest.in === paramTypes.PATH ? pathParams : queryParams ).push({ name, schema }) )
```